Classes/Controller/GetFileController.php

```diff
@@ -269,5 +274,12 @@ private function buildMetsXml($document)
 
         return $metsXml;
     }
+
+    private function isForbidden($action) {
```

Comment:
Only tiny hint: We should follow [PSR-2](https://www.php-fig.org/psr/psr-2/) with new code:

> Opening braces for methods MUST go on the next line, and closing braces MUST
> go on the next line after the body.